.sobelow-conf

```diff
@@ -0,0 +1,10 @@
+[
+  verbose: false,
+  private: false,
+  skip: true,
```

Comment:
Should we use `compact: true` The output looks like this

![image](https://user-images.githubusercontent.com/11751745/87289498-ecc53e00-c526-11ea-8bd5-156487f2b110.png)

compare with the non-compact mode

![image](https://user-images.githubusercontent.com/11751745/87289550-fea6e100-c526-11ea-8915-7532a765ad18.png)

Reply:
TBH, I don't have any preference on this, what do you guys think? @olivierobert @rosle @hanam1ni